python/felis/datamodel.py

```diff
@@ -93,18 +93,20 @@ class BaseObject(BaseModel):
     description: DescriptionStr | None = None
     """A description of the database object."""
 
-    @model_validator(mode="after")  # type: ignore[arg-type]
-    @classmethod
-    def check_description(cls, object: BaseObject, info: ValidationInfo) -> BaseObject:
+    votable_utype: str | None = Field(None, alias="votable:utype")
+    """The VOTable utype (usage-specific or unique type) of the object."""
+
+    @model_validator(mode="after")
```

Comment:
These changes are a fix to the Pydantic `model_validator` as it was using the wrong function signature.